sample/bin/clean.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
+rm /etc/nginx/nginx.conf
```

Comment:
Leftover... Also looks like the line below should be updated. Not sure if CodeDeploy will remove files previously added. Will test.